app/src/test/java/org/oppia/android/app/translation/AppLanguageResourceHandlerTest.kt

```diff
@@ -465,6 +473,16 @@ class AppLanguageResourceHandlerTest {
     assertThat(formattedString).containsMatch("[,.]")
   }
 
+  @Test
+  fun testToHumanReadableString_forInt_returnsStringWithExactNumber() {
```

Comment:
I suggest also testing this with non-English since it's meant to verify number localization (so we should make sure it does correctly localize the number).

Reply:
Code for testing in non-english (Arabic) has been added.